apps/server/tests/tasks/planTask.e2e-spec.ts

```diff
@@ -72,7 +72,169 @@ describe('PlanTask (e2e)', () => {
+      start: newDate(new Date(2022, 12, 21, 0)),
+      duration: moment.duration(9, 'hours'),
+      user: user,
+      priority: 'high',
+    });
+
+    await createConstTask({
+      taskName: 'Sleep',
+      category: sleepCategory,
+      // New date in utc
+      start: newDate(new Date(2022, 12, 22, 0)),
+      duration: moment.duration(12, 'hours'),
+      user: user,
+      priority: 'high',
+    });
+
+    await createConstTask({
+      taskName: 'Lectures',
+      category: lecturesCategory,
+      start: newDate(new Date(2022, 12, 20, 10, 0)),
+      duration: moment.duration(2, 'hours'),
+      user,
+    });
+
+    await createConstTask({
+      taskName: 'Lectures',
+      category: lecturesCategory,
+      start: newDate(new Date(2022, 12, 20, 12, 15)),
+      duration: moment.duration(705, 'minutes'),
+      user,
+    });
+
+    await createConstTask({
+      taskName: 'Lectures',
+      category: lecturesCategory,
+      start: newDate(new Date(2022, 12, 21, 10, 0)),
+      duration: moment.duration(2, 'hours'),
+      user,
+    });
+
+    await createConstTask({
+      taskName: 'Lectures',
+      category: lecturesCategory,
+      start: newDate(new Date(2022, 12, 21, 12, 15)),
+      duration: moment.duration(705, 'minutes'),
+      user,
+    });
+
+    await createConstTask({
+      taskName: 'Lectures',
+      category: lecturesCategory,
+      start: newDate(new Date(2022, 12, 21, 12, 15)),
+      duration: moment.duration(24, 'hours'),
+      user,
+    });
+
+    // Add float tasks
+
+    const yellowColor = await Color.create({
+      hexCode: '#ffff00',
+    }).save();
+
+    const preparationCategory = await Category.create({
+      name: 'Preparation',
+      color: yellowColor,
+      user: user,
+    }).save();
+
+    // Now let's simulate already planned float task:
+
+    const A = await Task.create({
+      name: 'A',
+      category: preparationCategory,
+      priority: 'high',
+      isFloat: true,
+      user: user,
+      chunkInfo: {
+        start: newDate(new Date(2022, 12, 20)),
+        minChunkDuration: moment.duration(15, 'minutes'),
+        maxChunkDuration: moment.duration(15, 'minutes'),
+        deadline: newDate(new Date(2022, 12, 22, 0, 0)),
+        estimation: moment.duration(30, 'minutes'),
+        chillTime: moment.duration(0, 'minutes'),
+      },
+    }).save();
+
+    const B = await Task.create({
+      name: 'B',
+      category: preparationCategory,
+      priority: 'low',
+      isFloat: true,
+      user: user,
+      chunkInfo: {
+        start: newDate(new Date(2022, 12, 20)),
+        minChunkDuration: moment.duration(60, 'minutes'),
+        maxChunkDuration: moment.duration(60, 'minutes'),
+        deadline: newDate(new Date(2022, 12, 23, 0, 0)),
+        estimation: moment.duration(120, 'minutes'),
+        chillTime: moment.duration(0, 'minutes'),
+      },
+    }).save();
+
+    await taskPlanner.planTask(A);
+    await taskPlanner.planTask(B);
```

Comment:
Can we assert desired state of this task chunks? (That would be actually testing things, not just checking if planTask not throwing anything)